frameit/lib/frameit/frame_converter.rb

```diff
@@ -64,6 +64,23 @@ def convert_frames
 
         UI.important "Converting PSD file '#{psd}'"
         image = MiniMagick::Image.open(psd)
+
+        if psd =~ /iPhone-SE/
+          UI.success "Removing white background 🚫 ⬜️"
+
+          good_layers = image.layers.reject do |layer|
+            label = layer.details['Properties']['label']
+            label =~ /White B/i || label.to_s.length == 0
```

Comment:
Is this rejecting layers with no labels? 

Maybe a comment in the code on how this works would be helpful